File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // init variables
    int n;

    // Prompt for height
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Print pyramid
    for (int i = 1; i <= n; i++)
    {
        // spaces before #
        for (int j = n; j > i; j--)
        {
            printf(" ");
        }

        // left #
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // center spaces
        printf("  ");

        // right spaces
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

h = 0
while (h < 1 or h > 8):
    h = get_int("Height: ")

for i in range(1, h+1):
    for j in range(h-i):
        print(" ", end = "")
    for cur in range(i):
        print("#", end = "")
    print("  ", end = "")
    for cur in range(i):
        print("#", end = "")
    print()
